### PR TITLE
Create outlines for csharp 7

### PR DESCRIPTION
**Pull request by BillWagner**
Add placeholders for new C# 7 topics.

### CODE REVIEW

**Comment by qinezh** [comment]
Open Publishing Build Service: The pull request content has been published and here are some changed files links of this time:
- Docset 1 - **core-docs**  
  [https://review.docs.microsoft.com/en-us/dotnet/articles/csharp/csharp-7?branch=pr-en-us-1115](https://review.docs.microsoft.com/en-us/dotnet/articles/csharp/csharp-7?branch=pr-en-us-1115)  
  [https://review.docs.microsoft.com/en-us/dotnet/articles/csharp/pattern-matching?branch=pr-en-us-1115](https://review.docs.microsoft.com/en-us/dotnet/articles/csharp/pattern-matching?branch=pr-en-us-1115)  
  [https://review.docs.microsoft.com/en-us/dotnet/articles/csharp/tuples?branch=pr-en-us-1115](https://review.docs.microsoft.com/en-us/dotnet/articles/csharp/tuples?branch=pr-en-us-1115)  
  Please see 1 more published files of this docset in report.  

**Status**: Succeeded.  
[Open Publishing Report](https://opbuildstorageprod.blob.core.windows.net/report/2016%5C10%5C4%5C2a032abb-a76c-bfc8-4231-71ab612989a2%5CPullRequest%5C201610041759595894-1115%5Cworkflow_report.html).

File: docs/csharp/whats-new.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ ms.assetid: 77deec51-a14d-46d4-9bb3-faf449477149
 
 
 * [C# 7](csharp-7.md):
-    - This page describes the latest features in the C# language. This covers C# 7, currently available in [Visual Studio 15 Preview 4](https://www.visualstudio.com/en-us/downloads/visual-studio-next-downloads-vs.aspx).
+    - This page describes the latest features in the C# language. This covers C# 7, currently available in [Visual Studio 15 Preview 4](https://www.visualstudio.com/visual-studio-pre-release-downloads/x).
```
Comment (mairaw) [inline docs/csharp/whats-new.md]: Should this link be just https://www.visualstudio.com/visual-studio-pre-release-downloads/? The x in the end takes you to a Xamarin page.

**Review by mairaw — COMMENTED** [review]
LGTM. I just had a question about a new link you introduced.

**Comment by qinezh** [comment]
Open Publishing Build Service: The pull request content has been published and here are some changed files links of this time:
- Docset 1 - **core-docs**  
  [https://review.docs.microsoft.com/en-us/dotnet/articles/csharp/csharp-7?branch=pr-en-us-1115](https://review.docs.microsoft.com/en-us/dotnet/articles/csharp/csharp-7?branch=pr-en-us-1115)  
  [https://review.docs.microsoft.com/en-us/dotnet/articles/csharp/pattern-matching?branch=pr-en-us-1115](https://review.docs.microsoft.com/en-us/dotnet/articles/csharp/pattern-matching?branch=pr-en-us-1115)  
  [https://review.docs.microsoft.com/en-us/dotnet/articles/csharp/tuples?branch=pr-en-us-1115](https://review.docs.microsoft.com/en-us/dotnet/articles/csharp/tuples?branch=pr-en-us-1115)  
  Please see 1 more published files of this docset in report.  

**Status**: SucceededWithWarning. **Message**: Warning occurred: Unable to resolve cross-reference "<a href="xref:Microsoft.Win32.SafeHandles.SafeNCryptKeyHandle" sourcefile="docs/standard/garbagecollection/implementing-dispose.md" sourcestartlinenumber="413" sourceendlinenumber="413">SafeNCryptKeyHandle</a>", "<a href="xref:Microsoft.Win32.SafeHandles.SafeNCryptProviderHandle" sourcefile="docs/standard/garbagecollection/implementing-dispose.md" sourcestartlinenumber="413" sourceendlinenumber="413">SafeNCryptProviderHandle</a>", "<a href="xref:Microsoft.Win32.SafeHandles.SafeNCryptSecretHandle" sourcefile="docs/standard/garbagecollection/implementing-dispose.md" sourcestartlinenumber="413" sourceendlinenumber="413">SafeNCryptSecretHandle</a>"..  
[Open Publishing Report](https://opbuildstorageprod.blob.core.windows.net/report/2016%5C10%5C6%5C2a032abb-a76c-bfc8-4231-71ab612989a2%5CPullRequest%5C201610061959077034-1115%5Cworkflow_report.html).

**Comment by qinezh** [comment]
Open Publishing Build Service: The pull request content has been published and here are some changed files links of this time:
- Docset 1 - **core-docs**  
  [https://review.docs.microsoft.com/en-us/dotnet/articles/csharp/csharp-7?branch=pr-en-us-1115](https://review.docs.microsoft.com/en-us/dotnet/articles/csharp/csharp-7?branch=pr-en-us-1115)  
  [https://review.docs.microsoft.com/en-us/dotnet/articles/csharp/pattern-matching?branch=pr-en-us-1115](https://review.docs.microsoft.com/en-us/dotnet/articles/csharp/pattern-matching?branch=pr-en-us-1115)  
  [https://review.docs.microsoft.com/en-us/dotnet/articles/csharp/tuples?branch=pr-en-us-1115](https://review.docs.microsoft.com/en-us/dotnet/articles/csharp/tuples?branch=pr-en-us-1115)  
  Please see 1 more published files of this docset in report.  

**Status**: SucceededWithWarning. **Message**: Warning occurred: Unable to resolve cross-reference "<a href="xref:Microsoft.Win32.SafeHandles.SafeNCryptKeyHandle" sourcefile="docs/standard/garbagecollection/implementing-dispose.md" sourcestartlinenumber="413" sourceendlinenumber="413">SafeNCryptKeyHandle</a>", "<a href="xref:Microsoft.Win32.SafeHandles.SafeNCryptProviderHandle" sourcefile="docs/standard/garbagecollection/implementing-dispose.md" sourcestartlinenumber="413" sourceendlinenumber="413">SafeNCryptProviderHandle</a>", "<a href="xref:Microsoft.Win32.SafeHandles.SafeNCryptSecretHandle" sourcefile="docs/standard/garbagecollection/implementing-dispose.md" sourcestartlinenumber="413" sourceendlinenumber="413">SafeNCryptSecretHandle</a>"..  
[Open Publishing Report](https://opbuildstorageprod.blob.core.windows.net/report/2016%5C10%5C6%5C2a032abb-a76c-bfc8-4231-71ab612989a2%5CPullRequest%5C201610062005591801-1115%5Cworkflow_report.html).

**Comment by rpetrusha** [comment]
LGTM.

**Pull request merged**